app/views/forms/_add_element.slim

```diff
@@ -1,6 +1,6 @@
 = render partial: 'shared/error_messages', locals: {target: form_element }
 
-= form_for [form, form_element], remote: true, html: {class: 'form-element'}  do |f|
+= form_for [form, form_element], remote: true, html: {class: 'form-element', id: 'new_collection_element'}  do |f|
```

Comment:
Me being pedantic, but let's use dashes, and not underscores for class names and ids.